### ISSUE TITLE
[Bug] [Might be inteded behavour] The mod appears to break 'render only current waypoint set'

### ISSUE THREAD

**Issue by 2001herne** [thebuildcraft/RemotePlayerWaypointsForXaero#32]
The mod appears to break the 'render only current waypoint set' option by forcing the worldmap to render all waypoint sets.  While I understand that rendering all waypoint sets presents a quick and easy way of not interfering with a users existing waypoints, the ability to hide specific waypoint sets is a critical feature of xaeros minimap/worldmap.

**Comment by thebuildcraft** [comment]
Yes, unfortunately, this is currently the intended behaviour as I didn't find a good way to force enable only the special sets the mods generates.
Maybe I will do it the "correct way" later, but I'm currently not so motivated to work on the mod... sorry.
